src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -205,6 +216,14 @@ def restart_completer(self):
             self.lexer = get_az_lexer(command_info)
         self._cli = None
 
+    def on_recommendation_prepared(self):
+        scenarios = self.recommender.get_scenarios() or []
+        scenarios_rec_info = "Scenario Recommendation: "
+        scenarios_rec_info += "".join([f'\n [{idx+1}] {s["scenario"]} ({len(s["nextCommandSet"])} Commands)' for idx, s in enumerate(scenarios)])
```

Comment:
Please simplify the complex expression into variables with better readability

Reply:
I'd like to confirm whether the `az config set interactive.enable_recommender=False` switch can control the closing of all the display of scenario recommendation

Reply:
> I'd like to confirm whether the `az config set interactive.enable_recommender=False` switch can control the closing of all the display of scenario recommendation

Yes. `interactive.enable_recommender` could controll most of the display of recommendation. However if `interactive.enable_recommender=False` is set with scenario recommendation layout on you will find an empty line below command description part.